script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt

```diff
@@ -171,22 +184,22 @@ That's right...
 [sha:e49f758b5079be250104615da13c7d5f3ebc17a1]{
 -- Page 397, Offset 34417.
 -- アレは本来、俺には手に入らないものだった。
-It really should've been out of my reach.
+By all rights, something like that should have been beyond my reach.
 }
 [sha:008c0d08107b1b737c95f8f3e866a854145764c1]{
 -- Page 397, Offset 34418.
 -- <未来|し>を恐れるだけの俺には見る事のできなかったもの。
-A sight that I'd never laid my eyes upon when all I did was live in fear of the <future|d e a t h>.
+Something that I, with my eyes clouded by fear of <the future|d e a t h       >, should never have been able to behold.
 }
 [sha:2d67b5ca2a4fa16617eba2d73de9f6f1e93b62f5]{
 -- Page 397, Offset 34419.
 -- 気付かないうちに見限っていた、何でもない光景だ。
-A commonplace scenery that I'd unconsciously given up on.
+The sort of prosaic scenery I'd unconsciously given up on.
 }
 [sha:929dbf24ad1445fe5d9fcb07cfd8f18c2343033e]{
 -- Page 398, Offset 34420.
 -- ……だから、いいんだ。
-...That's why, it's fine.
+That's why... I don't mind.
```

Comment:
Ehh, I kinda prefer it to So here, especially since it needs to get echoed a second time further away from the thing it's following on from, and `That's why` is a lot more clear on the causation.

Reply:
Maybe it's just that I like so...
Also that だから = "that's why" is a cliche and internal JPism for me, probably. Like, it's not actually a conjunction in proper English. It's used in sentences to seperate clauses (well, 'that' is actually what has that function') but rarely to start them. If it was...
>And that's why... I don't mind

It'd feel more proper, because "and" is a conjunction.